dev/bitcoin/rtl.tf

```diff
@@ -0,0 +1,14 @@
+resource "helm_release" "rtl" {
+  name       = "rtl"
+  chart      = "${path.module}/../../charts/rtl"
+  namespace  = kubernetes_namespace.bitcoin.metadata[0].name
+
+  dependency_update = true
+  values = [
+    file("${path.module}/rtl-values.yml")
```

Comment:
Can probably get rid of this since it's empty.

Reply:
cleaned out in [8e54598](https://github.com/GaloyMoney/charts/pull/1791/commits/8e5459805ad4e9fbdfadc00a6c9a5bda26e4b662)